tasks/facial_landmark_detection/train.py

```diff
@@ -148,6 +150,22 @@ def load_data(traindir, args):
     return dataset, dataset_test, train_sampler, test_sampler
 
 
+def setup_qat(model: nn.Module) -> None:
+    assert model is not None
+
+    _replace_relu(model)
+
+    model.eval()
+
+    # Modify quantization engine as appropriate for the target platform
+    model.setup_qconfig('fbgemm')
+    model.classifier.qconfig = None
+
+    model.fuse_model(is_qat=True)
```

Comment:
Is it really necessary to have the is_qat argument? It seems like we always call _fuse_model with True.